terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -1,22 +1,11 @@
 environment = "staging"
 aws_profile = "staging-eu-west-2"
 
-# scaling configs
-desired_task_count = 2
-
 # service configs
+use_set_environment_files = false
```

Comment:
I think if we want to read the .env file from s3 bucket, this variable to be set as true.